src/libDB/Archival.cpp

```diff
@@ -31,24 +31,14 @@ unsigned int REFRESH_DELAY = 5;
 
 void Archival::InitSync() {
   auto func = [this]() -> void {
-    m_synchronizer.FetchOfflineLookups(m_mediator.m_lookup);
+    if (!m_mediator.m_node->GetOfflineLookups()) {
```

Comment:
I dont think so we should "return" here as it will return from the whole sync thread then

Reply:
The success of this data fetching is very elementary for rejoining. We can adjust the times of retrying, but if it always fail, then I think it's better to terminate the rejoin first.

Reply:
but archival does not "rejoin", it is in pull mode

Reply:
I shall change the language, for "synchronizing"